zScanner/ViewModels/NewDocument/NewDocumentPhotosViewModel.swift

```diff
@@ -10,6 +10,21 @@ import UIKit
 import RxSwift
 import RxRelay
 
+class Picture: Equatable {
```

Comment:
Change name to PageDomainModel, put into a separated file, on a correct place in the folder structure

Reply:
use struct, not class until necessary

Reply:
It already exists so I added as a new attribute.